### DISCUSSION TITLE
Fahrpläne Erstellen

### DISCUSSION THREAD

**Discussion by zfFinn** [Traewelling/traewelling#2953] [Questions / Support] [answered]
Hallo zusammen,

würde die Möglichkeit bestehen für Museumsbahnen, Parkeisenbahnen,... eigene Fahrpläne zu hinterlegen (ohne Programmierkenntnisse) natürlich nur für Öffentliche Fahrten? Wenn nicht würde ich mich freuen wenn das Irgendwann möglich wäre und als kleiner Wunsch diesbezüglich: Man kann im Fahrplaninformationen hinterlegen die jeder sehen kann (bei jedem Zug) und Mehrere Fahrpläne an einem Tag zu erstellen um grade bei Parkeisenbahnen auf einen Schellen Fahrbahnwechsel sicher zu stellen da der aktuelle Fahrplan meistens erst Kurzfristig entschieden wird.

Ich finde die Idee von Träwelling sehr gut und auch super umgesetzt.

Gruß
Finn

**Comment by MrKrisKrisu** [comment]
Um welche Museumsbahnen, bzw. Parkeisenbahnen geht es denn hier konkret? Wäre es ggfs. eine Möglichkeit, dass ihr eure Fahrpläne an das [EFZ](https://de.wikipedia.org/wiki/Europ%C3%A4isches_Fahrplanzentrum) übermittelt? Dann sind diese Daten z.B. auch im DB Navigator verfügbar.

Träwelling nutzt für die Fahrplandaten nämlich lediglich das HAFAS der DB - welches generell das, was dem EFZ gemeldet wird kennt.

Es besteht technisch keine Möglichkeit, dass wir Fahrpläne "zwischenschleusen". Wenn es aber Haltestellen sind, die "uns" bekannt (aka im HAFAS verfügbar) sind, kann ein User auch [manuelle Fahrten anlegen](https://help.traewelling.de/features/manual-trips/).

**Comment by zfFinn** [comment]
Es wäre die Parkeisenbahn in der Wuhlheide (Berlin) und das Prignitzer Kleinbahnmuseum Lindenberg mit dem Pollo. Es ist nicht so einfach die Fahrpläne an das EFZ zu schicken da der tagesaktuell Fahrplan erst am Fahrtag um 9 Uhr entschieden wird. Vielen Dank für die schnelle Antwort. Ich kann mich ja mal mit dem EFZ beschäftigen. Gruß Finn

**Reply by HerrLevin** [reply]
Das Thema "Fahrpläne für Parkeisenbahnen" wollte ich tatsächlich schon mal angehen, das ist aber kein allzu leichtes Unterfangen - zudem mir da noch etwas der Einblick in die Abläufe der Gestaltung eines Fahrplans fehlt. Falls du da ein paar weitere Infos für mich hast, was ihr da so in etwa braucht, schick mir gerne ne Mail an levin@traewelling.de

**Comment by MrKrisKrisu** [comment, marked as answer]
Als neues Update an diese Stelle:

Da wir jetzt auf Transitous umgestiegen sind wäre eine Option, wenn Unternehmen/Vereine, die regelmäßige Fahrten durchführen z.B. eine GTFS bereitstellen (und auch pflegen!). Dann kann man diese in Transitous einbinden und die Fahrten sind dann auch bei Träwelling verfügbar.

**Comment by zfFinn** [comment]
Danke für's Update.Was ist denn GTFS?--Grüße aus Portugal Finn Am 28.04.25, 20:36 schrieb Kris ***@***.***>:

Als neues Update an diese Stelle:
Da wir jetzt auf Transitous umgestiegen sind wäre eine Option, wenn Unternehmen/Vereine, die regelmäßige Fahrten durchführen z.B. eine GTFS bereitstellen (und auch pflegen!). Dann kann man diese in Transitous einbinden und die Fahrten sind dann auch bei Träwelling verfügbar.
